src/users/data/api.js

```diff
@@ -542,7 +542,7 @@ export async function postTogglePasswordStatus(user, comment) {
 
 export async function postResetPassword(email) {
   try {
-    const { data } = await getAuthenticatedHttpClient().post(AppUrls.getResetPasswordUrl(), `email_from_support_tools=${email}`);
+    const { data } = await getAuthenticatedHttpClient().post(AppUrls.getResetPasswordUrl(), `email_from_support_tools=${encodeURIComponent(email)}`);
```

Comment:
Instead of sending a string and working with encodings directly, would it work to pass it as a dictionary? I can't tell just from a quick glance at the docs, but I suspect Axios will do the right thing here.

```suggestion
    const { data } = await getAuthenticatedHttpClient().post(AppUrls.getResetPasswordUrl(), {email_from_support_tools: email});
```

Reply:
You're right — sending the data as a JSON object is generally more standard with Axios, and it would work well under most circumstances. However, in our case, the password_change_request_handler API is a @require_POST Django view that reads data from request.POST. This means it's expecting application/x-www-form-urlencoded data (i.e., a URL-encoded string), not a JSON body.

Changing the frontend to send a JSON object would require corresponding updates on the backend to parse request.body as JSON instead of relying on request.POST. Since this API is used in multiple places, making that change would require updating and testing each usage to ensure we don't break any existing flows.

Reply:
Ah yeah, it probably defaults to sending JSON, doesn't it -- so you'd have to tell it to use `application/x-www-form-urlencoded`. https://axios-http.com/docs/post_example gives some examples of that.

Reply:
I think we should use Axios object syntax with URL-encoded data. Its was a  great suggestion.